index.html

```diff
@@ -21,17 +22,20 @@
         <nav>
             <a class="logo-item" href="index.html">Peter chendi</a>
 
-            <img class="handberga" src="Images/Icons/Union.png" alt="menu">
-            <ul>
-                <li><a>Portfolio</a></li>
-                <li><a>About</a></li>
-                <li><a>Contact</a></li>
-                <li><a><img src="Images/Icons/Button/Tertiary/Icons/mail.png" alt="email icon"></a></li>
+            <img id="handberga-menu"  class="handberga" src="Images/Icons/Union.png" alt="menu">
+        
+            <ul id="menu">
+                <li class="menu-list"><i id="exit" class="bi bi-x-lg"></i></li>
+                <li><a href="#home">Portfolio</a></li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#contact">Contact</a></li>
+                <li class="image-link"><a><img  src="Images/Icons/Button/Tertiary/Icons/mail.png" alt="email icon"></a></li>
             </ul>
```

Comment:
- [ ] Great job so far, one of the requirements is to make sure that the mobile menu doesn't appear on desktop devices, kindly fix that.

![Screen Shot 2022-09-06 at 11 18 22 AM](https://user-images.githubusercontent.com/59416136/188583909-ae8576b3-434f-48cc-9b9e-adf76b81ccc1.png)